src/filesink.cpp

```diff
@@ -54,12 +54,12 @@ namespace g3 {
       out << message.get().toString() << std::flush;
    }
 
-   std::string FileSink::changeLogFile(const std::string &directory) {
+   std::string FileSink::changeLogFile(const std::string &directory, const std::string &logger_id) {
```

Comment:
I see that it is useful to specify the logger ID in the constructor. 

What is the use case and need that you see to specify it when changing the log file?